src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -325,7 +325,8 @@ class ShenandoahFreeSet : public CHeapObj<mtGC> {
   // hold evacuated objects.  If this occurs and memory is still available in the Mutator's free set, we will flip a region from
   // the Mutator free set into the Collector or OldCollector free set.
   void flip_to_gc(ShenandoahHeapRegion* r);
```

Comment:
It seems as if (the current implementation of) `flip_to_gc()` always succeeds in flipping. I'd add that to its spec comment.